src/main/java/net/dv8tion/jda/core/requests/RestAction.java

```diff
@@ -679,4 +739,44 @@ public T complete(boolean shouldQueue)
         @Override
         protected void handleResponse(Response response, Request<T> request) { }
     }
+    /**
+     * Specialized form of {@link net.dv8tion.jda.core.requests.RestAction} that is used to provide information that
+     * has already been retrieved or generated so that another request does not need to be made to Discord.
+     * <br>Basically: Allows you to provide a value directly to the success returns.
+     *
+     * @param <T>
+     *        The generic response type for this RestAction
+     */
+    public static class FailedRestAction<T> extends RestAction<T>
```

Comment:
This javadoc is misleading 
Did you also add one of these to `AuditableRestAction`?